v2/devdoc/log_context_property_type_if_requirements.md

```diff
@@ -6,6 +6,8 @@
 - copy
 - free
 - get_type
+- init
+- get_init_data_size
```

Comment:
should these 2 be added to  struct LOG_CONTEXT_PROPERTY_TYPE_IF_TAG? Between lines 25 and 26?

Reply:
Apparently they do not, looking in all the code.

Reply:
Yep.